lib/osi-licenses.yml

```diff
@@ -0,0 +1,54 @@
+# Generated by ChatGPT on 2023-11-18
+# https://chat.openai.com/c/9785e91f-37ab-4d13-8221-8e1bb10e8bc1
```

Comment:
This URL is not accessible without a login. I did not test if it is accessible with a login.